### PR TITLE
feat: set up PR to change RPC to Partners

### PR DESCRIPTION
**Pull request by elPiablo**
## Reference issue: 
- https://github.com/Developer-DAO/academy-turbo/issues/93#issue-2097133538

## Changes
- [x] overhaul RPC Fundamentals lesson by replacing textual mentions, images and links of RPCs (and Metamask) to https://github.com/Developer-DAO/academy-turbo/issues/93#issuecomment-1907069250

### CODE REVIEW

**Comment by elPiablo** [comment]
@kempsterrrr could you review this PR: https://github.com/Developer-DAO/academy-turbo/pull/101 , please?
I understand you want to gather the clicks, there's just some conflicting, or additional messy steps involved then regarding the Perks link you provided: https://devdao.to/grove-perk 

What am I talking about?

On Academy, we always encourage devs to create a separate browser profile and a separate wallet for dev work without any valuable tokens in them. Bear in mind, most of what we are expecting learners at this stage is **not** so readily to be deploying contracts for apps on a Mainnet with valuable tokens. 

The Grove Perks link is expecting you to have a D4R or 400 CODE, which would then involve a Mainnet tx with those tokens from a user's D_D wallet to their 'dev wallet' to align with our platform advice.

So far, in the RPC lesson, I suggested that the Grove Perks link be for D_D members,  I gave an alternative to non-D_D heads to use the Nodies link: https://www.nodies.app/. As I understood from the POKT site, that Grove is a paid service, and there is a free tier for Nodies, is that right?

Bottom line, we need to resolve this 'developer wallet' and RPC sign up with Perks. I would like to know what steps to give learners across multiple lessons and get that rectified. Cheers in advance for jumping in with this.

I'll just drop this same post in the PR, so the lads can see where I'm at with it.

**Comment by kempsterrrr** [comment]
Thanks for pushing this forward @elPiablo

Very good callout here regarding the flow. Grove does has a free tier which you can see on their [pricing page](https://www.grove.city/pricing), what seems to be breaking the flow here is the process to redeem benefits as it assumes users are already DAO member which in most cases is likely not to be true.

Suggest updating the language to recommend Grove or Nodies, and then adding a note to say if you're a developer dao member you can get a discount for these services here - and provide a link to the benefits page [here](https://devdao.to/member-benefits). 

For Zerion wallet sign-up they wont need to go via perks. This link will take them directly to install the wallet - https://devdao.to/get-zerion (lmk if I'm missing something here or if this is not ok)

As a general piece of feedback, it took me a few takes to find the relevant content on this page. 

Some points to consider:

- Give links their own styling which clearly shows they're links
- Consider breaking up sections a little further by either using bullet points or more sub-headings. There is a lot of text in this block below, might be worth adding section like: "What RPC to use" or "Recommended RPCs" - the bullet point approach on the last version of this lesson [here](https://academy.developerdao.com/lessons/fundamentals/connect-with-rpc) I personally find easier to read.

image

**Comment by elPiablo** [comment]
> Very good callout here regarding the flow. Grove does has a free tier which you can see on their [pricing page](https://www.grove.city/pricing), what seems to be breaking the flow here is the process to redeem benefits as it assumes users are already DAO member which in most cases is likely not to be true.
> 
> Suggest updating the language to recommend Grove or Nodies, and then adding a note to say if you're a developer dao member you can get a discount for these services here - and provide a link to the benefits page [here](https://devdao.to/member-benefits).

will give this a good shot.  cheers.

> For Zerion wallet sign-up they wont need to go via perks. This link will take them directly to install the wallet - https://devdao.to/get-zerion (lmk if I'm missing something here or if this is not ok)

this 'lesson' had been done. Will need some polish re: bullet pointing steps, but I was waiting to see how the RPC one looks first, and then try and create a 'predictable format' for these types of things, and come back and polish. (https://academyv2.vercel.app/tracks/fundamentals/wallets)

> * Give links their own styling which clearly shows they're links
Done now!! woo hoo!

> * Consider breaking up sections a little further by either using bullet points or more sub-headings. There is a lot of text in this block below, might be worth adding section like: "What RPC to use" or "Recommended RPCs" - the bullet point approach on the last version of this lesson [here](https://academy.developerdao.com/lessons/fundamentals/connect-with-rpc) I personally find easier to read.

In the pipeline, cheers!

**Review by Markkos89 — APPROVED** [review]
